site/themes/arangodb-docs-theme/static/css/theme.css

```diff
@@ -144,6 +144,44 @@ button, input, optgroup, select, textarea {
     margin-bottom: 15px;
 }
 
+.button {
+    background: transparent;
+}
+
+.button__wrapper {
+    border-radius: 4px;
+    text-align: center;
+    text-decoration: none;
+    border: none;
+    cursor: pointer;
+    width: fit-content;
+    height: 2rem;
+}
+
+.button__wrapper__green {
```

Comment:
Instead of such big names.. we can say `button.primary` and use it as `button primary`. Similar for others

Reply:
Well contextually.. can be `button.green` or `button.red` also. However, a more non-implicit naming convention like above is easier to understand and use it consistently.